cyberint-argos/README.md

```diff
@@ -1,19 +1,18 @@
-
 # Cyberint-Argos API connector for QRadar
 
-# How to Generate Cyberint API token
+## How to Generate Cyberint API token
 
 1. Log on to your Cyberint instance through: https://\<MyCompuny>/cyberint.io.
 
 2. At the top right of the screen press the profile button and then enter the "USER SETTINGS".
 
-3. In the user settings press at three doted in the top right.
+3. In the user settings press at three dots in the top right.
```

Comment:
Not that it is needed to change, because what you wrote might be clearer.
But just letting you know that the icon has a name:
![image](https://github.com/qmasters-ltd/qradar-universal-cloud-rest-api/assets/106475467/afb0887b-688d-46fb-ac23-c50ff0ce39d8)